src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,11 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
+
+    public String getFeedbackToUser() {
```

Comment:
Good that you encapsulated this. However, do you think it is really necessary?

Hint: Look at the original declaration.